spec/src/lib.rs

```diff
@@ -200,6 +200,54 @@ impl Genesis {
             .transaction(cellbase_transaction)
             .transaction(dep_group_transaction)
             .build();
+
+        // Check lock scripts
```

Comment:
It's better to put this part in a single function.

Reply:
make sense